### ISSUE TITLE
SVT:Appsody templates must be disabled in codewind che for 0.4 release

### ISSUE THREAD

**Issue by jagraj** [eclipse/codewind#444]
**Codewind version:** 0.4
**OS:** RedHat

**Che version:** 7.1
**IDE extension version:** 0.4
**IDE version:**  Theia
**Kubernetes cluster:** RedHat openshift

**Description:**
Appsody projects must be disabled for 0.4 release and we do not support them in eclipse che.

**Steps to reproduce:**
1. Install codewind 0.4 on OpenShift.
2. Select which templates to enable.
3. Select Kabanero template repo.

With the above steps user can create Kabanero templates and these should be disabled. 

![image](https://user-images.githubusercontent.com/713849/64871074-7f990180-d612-11e9-975c-b314d1f29d6f.png)

**Workaround:**

**Comment by tetchel** [comment]
@stalleyj can Portal not add the Kabanero repo when initializing the `repository_list` on Che?

**Comment by deboer-tim** [comment]
Sorry, this was my bad not thinking through the temporary state we're in with Che. Portal, do we have a simple way of doing this only on local for now? If not, it may be better to delay this until the current iteration.

**Comment by tobespc** [comment]
We don't have a simple way of doing this but I guess we could look at doing something clever at install time

**Comment by tobespc** [comment]
PR's are in, closing

**Comment by jagraj** [comment]
@tobespc 

I moved to latest pfe image and I still see option for the users to select Kabanero collections.

![Screen Shot 2019-09-18 at 10 26 56 AM](https://user-images.githubusercontent.com/713849/65157524-de89bc80-d9fe-11e9-9a6c-ee312f5b9b1d.png)

**Comment by tetchel** [comment]
I see this too. It also appears that the kabanero source is not marked as "protected" [code](https://github.com/eclipse/codewind/blob/master/src/pfe/portal/modules/Templates.js)
```
const KABANERO_REPO = {
  url: 'https://github.com/kabanero-io/collections/releases/download/v0.1.2/kabanero-index.json',
  description: 'Kabanero Collections',
  enabled: false,
};
```

**Comment by sujeilyfonseca** [comment]
I'm also seeing this problem with Codewind for Eclipse Che (OKD/Openshift Cluster).

<img width="595" alt="Screen Shot 2019-09-18 at 1 10 55 PM" src="https://user-images.githubusercontent.com/49531960/65170052-dbe69180-da15-11e9-9883-c7f3b1508f6d.png">

**Comment by jagraj** [comment]
Looks like you are right fix has been delivered to master and release branch. We were keep seeing the problem due to workspace data persisted when we created workspace with older version of codewind 0.4. We were just stoping and restarting workspace to get updates to the images. To see the change user has to delete the workspace and create new workspace. (https://github.com/eclipse/codewind/issues/444) has been resolved and it is closed now.
